### DISCUSSION TITLE
★復習★課題01~03までのについて

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#58] [Q&A]
01~03までをプッシュしたのですが、なぜかgitHubの方でプルリクエストが表示されることがありませんでした。
VScode上で、再度プッシュしてみるとプッシュが完了しているとメッセージが出るので、できているはずです・・。

プルリクエストは出せていますでしょうか？

**Comment by nsc-kiwamu** [comment]
プルリクエストは出ていないですね。。。
